src/utility-types.ts

```diff
@@ -43,7 +43,7 @@ export interface MessageValue<
   /**
    * Kill code.
    */
-  readonly kill?: KillBehavior | number
+  readonly kill?: KillBehavior | 1
```

Comment:
I think I can remove the `1` because
1. we already changed the default
2. it doesn't survive the check as before :warning: 

> const isKillBehaviorOptionHard = message.kill === killBehaviorTypes.HARD
> `isKillBehaviorOptionHard` was `false` if `message.kill` contains `1` :warning:

Reply:
1 is still needed since that we use it to send a message back to the worker to clearInterval and destroy the async context.
This can be changed later we don't need to do everything now :)